### ISSUE TITLE
problemn with visualization

### ISSUE THREAD

**Issue by hjonathan2019** [MiningCryptoLive/miningcore-web-ui#1]
hello, this is a great proyect!
 i have one visualizacion issue, in PAYMENTS tab... 
this is the way it looks... any way to fix it?
![image](https://user-images.githubusercontent.com/59586038/198078374-2e4c95f5-2df8-425a-9dac-78149f01e0c4.png)

**Comment by hjonathan2019** [comment]
@MiningCryptoLive

**Comment by MiningCryptoLive** [comment]
I did an update to it. Do a git pull and see if it looks better. If you want to modify it more you need to go to assets/js/miningcore-ui.js line 456.

**Comment by hjonathan2019** [comment]
@MiningCryptoLive what is a git pull?

**Comment by MiningCryptoLive** [comment]
Did you clone it to the server? If you did, go in the folder and run git pull to pull down the changes. If you downloaded it then just download it again. Or you can just copy miningcore-ui.js to your server.

**Comment by hjonathan2019** [comment]
@MiningCryptoLive i downloaded it last night... should i download it again now then?

**Comment by MiningCryptoLive** [comment]
Yes download it again and it has the changes I just made. Let me know if it looks better. I'm at work so I can't test it

**Comment by hjonathan2019** [comment]
@MiningCryptoLive looks like charm. love you!!

**Comment by MiningCryptoLive** [comment]
What's your pool address?

**Comment by hjonathan2019** [comment]
@MiningCryptoLive i loose the tittles, but i can live with that, unless it was easy to add it 
![image](https://user-images.githubusercontent.com/59586038/198090503-44b3dcc8-5313-4664-b7a5-6a575cc1f5fb.png)

**Comment by MiningCryptoLive** [comment]
> What's your pool address?

@hjonathan2019

**Comment by hjonathan2019** [comment]
erg-latam-pool.com.ar

**Comment by MiningCryptoLive** [comment]
Oops. I can add titles tonight after work and let you know

**Comment by hjonathan2019** [comment]
@MiningCryptoLive ok meanwhile i will try to do it myself

**Comment by hjonathan2019** [comment]
@MiningCryptoLive did it
![image](https://user-images.githubusercontent.com/59586038/198093489-9973a93e-1778-4a91-bfed-313a7a919947.png)

**Comment by hjonathan2019** [comment]
i will leave here how i did it so maybe can help in future

function loadPaymentsList() {
  	return $.ajax(API + "pools/" + currentPool + "/payments?page=0&pageSize=500")
	.done(function(data) {
	var paymentList = '<thead><tr><th rowspan="2">Fecha</th><th>Direccion</th><th>Monto</th><th>Confirmacion</th></tr></thead><tbody>';

![image](https://user-images.githubusercontent.com/59586038/198093893-c632fda2-1149-4fec-9a46-4700b5d98488.png)

**Comment by hjonathan2019** [comment]
@MiningCryptoLive i have a weird problem...
i put my wallet (dashboard tab), and only show me 1 miner in the list, and i have 5 in total... 
but web shows me total hashrate,,,,, super weird

![image](https://user-images.githubusercontent.com/59586038/198119826-a7985d56-6b99-454f-98ad-0461f7c6a303.png)

**Comment by MiningCryptoLive** [comment]
I am assuming they are all using the same wallet? What mining software are you using?
